README.md

```diff
@@ -1,2 +1,75 @@
-# tmail-ui-user
-This project is a user interface Flutter app for [TMail back-end](https://github.com/linagora/tmail-backend)
+# Tmail Flutter mobile application
```

Comment:
We should provide the information related to https://github.com/linagora/jmap-dart-client also. Am I right?

Reply:
I would qualify this a technical detail but ok